querydsl-core/src/main/java/com/querydsl/core/support/CollectionAnyVisitor.java

```diff
@@ -110,7 +110,7 @@ protected Predicate exists(Context c, Predicate condition) {
             Path<?> parent = (Path<?>) expr.getMetadata().getParent().accept(this, context);
             expr = new PathImpl<Object>(expr.getType(), PathMetadataFactory.forCollectionAny(parent));
             EntityPath<?> replacement = new EntityPathBase<Object>(expr.getType(),
-                    ExpressionUtils.createRootVariable(expr));
+                    ExpressionUtils.createRootVariable(expr, replacedCounter++));
```

Comment:
@timowest just a coding style question.
Is this statement okay for you? Most coding styles suggest that one statement (one line) should do one thing.
I was wondering what your opinion is, so that we can incorporate this into checkstyle (I think it has rules for that).